pennylane/transforms/hamiltonian_expand.py

```diff
@@ -15,10 +15,15 @@
 Contains the hamiltonian expand tape transform
 """
 # pylint: disable=protected-access
+from typing import Sequence
+
 import pennylane as qml
+from pennylane.measurements import Expectation
+from pennylane.ops import Sum
+from pennylane.tape import QuantumScript
 
 
-def hamiltonian_expand(tape, group=True):
+def hamiltonian_expand(tape: QuantumScript, group=True):
```

Comment:
Out of curiosity: why is tape declared (is that the correct term) but group not?

Reply:
Giving a default value to an argument already defines its type:
<img width="222" alt="image" src="https://user-images.githubusercontent.com/49232747/199728399-3d696c7f-44d3-4642-b356-1eb77da649c6.png">

Reply:
thanks I didnt know that! :)